interop-cats/shared/src/test/scala/zio/interop/catzQueueSpec.scala

```diff
@@ -0,0 +1,99 @@
+package zio.interop
+
+import cats.effect.implicits._
+import cats.effect.{ Effect, Concurrent, ContextShift, IO => CIO }
+import cats.implicits._
+import zio.test.Assertion._
+import zio.test._
+import zio.test.interop.catz.test._
+import zio.{ DefaultRuntime, Runtime, ZEnv }
+
+import scala.concurrent.ExecutionContext.global
+
+object catzQueueSpec
+    extends DefaultRunnableSpec({
+      def boundedQueueTest[F[+_]](implicit F: Concurrent[F], R: Runtime[ZEnv]): F[TestResult] =
+        for {
+          q  <- Queue.bounded[F, Any, Int](1)
+          _  <- q.offer(1)
+          f  <- q.offer(2).start
```

Comment:
This is going to be flaky because the forked fiber could offer the second value to the queue before `takeAll`

Reply:
This queue is bounded at size 1 and it’s full after `offer(1)` so it will suspend `offer(2)` and it won’t complete unless a `takeAll` is done. I actually purposely did this to test suspension although I’m not sure if this is the he greatest way to do it

Reply:
This queue is bounded at size 1 and it’s full after `offer(1)` so it will suspend `offer(2)` and it won’t complete unless a `takeAll` is done. I actually purposely did this to test suspension although I’m not sure if this is the he greatest way to do it

Reply:
Ah, missed that, sorry!